### DISCUSSION TITLE
Web access via own Nextcloud

### DISCUSSION THREAD

**Discussion by DiSaNet** [saber-notes/saber#1230] [General]
Hi Guys 
I'm new here and so far I'm quite excited about this project. I'm testing saber for my needs at the moment with my own NC using Linux and Android. That works perfect. But sometimes there is no chance to use my systems to call up notes so to make all my dreams come true for notes management is there a posibility to call them up on me NC to see and modify them or to show the Whiteboard? That would make it work for me.
THX
D

**Comment by ceskyDJ** [comment]
There was a web version at the beginning of the development. It was too hard to maintain, so the web support (and corresponding Nextcloud app support, which could be introduced later) had been abandoned. You can still access your notes via desktop and mobile apps for most platforms, so I hope there is a way how to handle your situation.
